popdat/src/import_census.rs

```diff
@@ -1,11 +1,126 @@
-use map_model::Map;
-
 use crate::CensusArea;
+use abstutil::Timer;
+use geo::algorithm::intersects::Intersects;
+use geojson::GeoJson;
+use map_model::Map;
+use std::convert::TryFrom;
 
 impl CensusArea {
-    pub fn find_data_for_map(_map: &Map) -> Result<Vec<CensusArea>, String> {
-        // TODO importer/src/utils.rs has a download() helper that we could copy here. (And later
-        // dedupe, after deciding how this crate will integrate with the importer)
-        todo!()
+    pub fn find_data_for_map(map: &Map, timer: &mut Timer) -> anyhow::Result<Vec<CensusArea>> {
+        // TODO eventually it'd be nice to lazily download the info needed. For now we expect a
+        // prepared geojson file to exist in data/system/<city>/population_areas.geojson
```

Comment:
`system` makes sense to me, because we'll need this at runtime. (Just thinking aloud)